Bottom: 646b12b3cf671d83e2dd12febbca1c8f6ceb54bc
Top:    73a38369f7fcf42f8612eab528a604ea4a6a16ab
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-19 15:41:27 +0800

Refresh of autofs-5.0.7-allow-use-of-hosts-map-in-maps.patch

---

diff --git a/modules/parse_sun.c b/modules/parse_sun.c
index 9aac7b5..7fd1bca 100644
--- a/modules/parse_sun.c
+++ b/modules/parse_sun.c
@@ -1603,9 +1603,12 @@ int parse_mount(struct autofs_point *ap, const char *name,
 			p = skipspace(p);
 		}
 
-		/* if it's not a hosts map loc must be non-null */
+		/*
+		 * If it's a hosts map loc will be empty but may contain
+		 * junk.
+		 */
 		if ((strstr(options, "fstype=autofs") &&
-		      strstr(options, "hosts"))) {
+		     strstr(options, "hosts"))) {
 			loc = NULL;
 			loclen = 0;
 		} else {
